Added tip section for form components (#6890)
yamada-ui/yamada-ui: www/contents/components/(components)/input.mdx

</InputGroup.Root>
```

:::tip

> To select a file, use [FileInput](/docs/components/file-input).

Reviewer: Please align this wording with the existing tips and keep the condition concise.

```suggestion
To let users select a file, use [FileInput](/docs/components/file-input).
```

:::

:::tip

> For password input, use [PasswordInput](/docs/components/password-input).

Reviewer: Please align this wording with the existing tips and keep the condition concise.

```suggestion
To let users enter a password, use [PasswordInput](/docs/components/password-input).
```

:::

:::tip

> For entering PIN codes or OTPs, use [PinInput](/docs/components/pin-input).

Reviewer: Please align this wording with the existing tips and keep the condition concise.

```suggestion
To let users enter a PIN code or OTP, use [PinInput](/docs/components/pin-input).
```

:::

:::tip

> For numeric input, use [NumberInput](/docs/components/number-input).

Reviewer: Please align this wording with the existing tips and keep the condition concise.

```suggestion
To let users enter a number, use [NumberInput](/docs/components/number-input).
```

:::

:::tip  

> For multi-line text input, use [Textarea](/docs/components/textarea).  

Reviewer: Please align this wording with the existing tips and keep the condition concise.

```suggestion
To let users enter multi-line text, use [Textarea](/docs/components/textarea).
```

:::

:::tip
インラインで編集可能なテキスト入力には、[Editable](/docs/components/editable)を使用します。

> :::

Reviewer 2: This looks like two Japanese tips were pasted into the English page. To keep the EN/JA pair aligned with the rest of the docs, I think these two tips should be translated to English.

```suggestion
:::tip
For multi-line text input, use [Textarea](/docs/components/textarea).
:::

:::tip
For inline editable text input, use [Editable](/docs/components/editable).
:::
```

Author: Fixed.

> :::

Author: Fixed.

> For inline editable text input, use [Editable](/docs/components/editable).  

Reviewer: Please align this wording with the existing tips and keep the condition concise.

```suggestion
To let users edit text inline, use [Editable](/docs/components/editable).
```
